.travis.yml

```diff
@@ -1,7 +1,7 @@
 # author: apolo.yasuda@ge.com
 
 language: minimal
-if: head_branch =~ ^v1.1beta_helm_pkg_update$ OR branch = v1.1beta_helm_pkg
+if: head_branch =~ ^v1.1beta_helm_pkg_update$ OR branch = v1.1beta_helm_pkg_update_multi_contnr_agent_lber
```

Comment:
this shouldn't be modified. @

Reply:
Branch name updated. However, if I update the branch name to `v1.1beta_helm_pkg`, its not triggering the CI.